src/abi/composableStablePoolV5.ts

```diff
@@ -0,0 +1,840 @@
+export const composabableStablePoolV5Abi = [
+    [
```

Comment:
There extra brackets here that should be removed.

Reply:
done